git.json

```diff
@@ -29,13 +29,13 @@
         "repo_uri": {
             "description": "Repo URI",
             "data_type": "string",
-            "required": true,
+            "required": false,
```

Comment:
Seems like `repo_uri` is a required parameter as we will need this `uri` to check the connectivity, without this asset will not be configured.